CODE_OF_CONDUCT.md

```diff
@@ -2,24 +2,26 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion,  sexual identity and orientation, or other factors.
 
 ## Our Standards
 
-Examples of behavior that contributes to creating a positive environment include:
+Examples of behavior which contributes to creating a positive environment include:
```

Comment:
I think "that" is correct, as I believe this is a [defining clause](https://www.grammarly.com/blog/which-vs-that/)

Reply:
@oddlittlebird ?

Reply:
Both are nominally correct, I prefer "that," because it is less ambiguous. When in doubt, refer to the Microsoft Style Guide: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which